src/Sign.Cli/StandardStreamWriterExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+using System.CommandLine;
+using System.CommandLine.IO;
+using System.Globalization;
+
+namespace Sign.Cli
+{
+    internal static class StandardStreamWriterExtensions
+    {
+        internal static void WriteFormattedLine(this IStandardStreamWriter writer, string format, params IdentifierSymbol[] symbols)
+        {
+            string[] formattedSymbols = symbols
+                .Select(symbol => $"--{symbol.Name}")
+                .ToArray();
+
+            writer.WriteLine(string.Format(CultureInfo.CurrentCulture, format, formattedSymbols));
```

Comment:
Shouldn't this be `InvariantCulture` as these are parameters and not user input?

Reply:
Please, in this case

Reply:
I have rebased the PR and update the commit to include this change.

Reply:
Looks like it's still showing CurrentCulture?